packages/realm-web/src/App.test.ts

```diff
@@ -0,0 +1,211 @@
+import { expect } from "chai";
+
+import { App } from "./App";
+import { User, UserState } from "./User";
+import { Credentials } from "./Credentials/index";
+import { MockNetworkTransport } from "./test/MockNetworkTransport";
+
+describe("App", () => {
+    it("can call the App as a constructor", () => {
+        const app = new App("default-app-id");
```

Comment:
Maybe the API should be `new App({ id: "default-app-id" });`

Reply:
I've chosen this pattern because it was mentioned in the "User-Facing Syntax" section of the "Scope: Merging Stitch/Realm SDKs" document.

Reply:
Java has two constructors, one that just take the String app Id for the simple, default case, and one taking a `RealmAppConfiguration` with a number of config options: https://github.com/realm/realm-java/pull/6788/files#diff-f620e2241952332f27e6ec1a7325860bR51

Reply:
@cmelchior that could work as well. What does Cocoa do @jsflax / @leemaguire ?

Reply:
I like the @cmelchior 's suggestion.